base/templates/base/base.html

```diff
@@ -1,129 +1,131 @@
+{% load static %}
+{% load bootstrap3 %}
 <!DOCTYPE html>
-<html>
-    
-    <head>
-        {% load bootstrap3 %}
-        {% bootstrap_css %}
-        {% bootstrap_javascript jquery=1%}
-        {% load staticfiles %}
-        {% block before-header %}{% endblock %}
-        <script type="text/javascript" src="{% static '/js/jsi18n'%}"></script>
-        <link rel="shortcut icon" href="{% static 'favicon.ico' %}"  type="image/x-icon" />
-        <title>
-            OEP{% block title %}{% endblock %}
-        </title>
-        <link rel="stylesheet" href="{% static 'css/theme-style.css' %}">
-        <link rel="stylesheet" href="{% static 'css/base-style.css' %}">
-        {% block after-header %}{% endblock %}
-    </head>
-
+<html lang="en">
```

Comment:
I added `lang="en"` for defining the used language, this makes sense if we have one primary language.

If we go one day i18n, this should be changed, but should be done by either using subdomains "`en.|de.|fr.|...`" or "`domainname/de-DE|domainname/fr-FR|domainname/en-US|...`". Because search engines don't like websites that change their content (language texts) on the same URL magically ...